core/chains/evm/logpoller/integration_test.go

```diff
@@ -161,3 +167,46 @@ func TestLogPoller_Integration(t *testing.T) {
 
 	require.NoError(t, lp.Close())
 }
+
+func TestLogWritesScale(t *testing.T) {
+	// Measure log write time.
+	t.Skip()
```

Comment:
Why is it skipped?

Reply:
its just code for running a local scale test